### ISSUE TITLE
[feature request] mindar-ui-scanning when target lost

### ISSUE THREAD

**Issue by marcusx2** [hiukim/mind-ar-js#309]
I like the mindar-ui-scanning overlay, but it only shows up before the very first time the target is found. If the user moves away from the target and it becomes lost, it doesn't come back. Can an option be added so that I can easily call the overlay when the target is lost? I think it makes sense to always have the scanning overlay when there is no target being tracked to tell the user to look for the target.

**Comment by marcusx2** [comment]
As a workaround, I managed to do it like this

```
var targetLostCallback = function() {
document.querySelector('.mindar-ui-scanning')?.classList.remove('hidden');
};
```

When the target is found, the ui-scanning is removed automatically, but when the target is lost it runs the callback above to show the ui-scanning overlay again, by removing the 'hidden' class.

**Comment by marcusx2** [comment]
As I explained, you can run the code snippet I gave as a callback to the targetLost event. Removing the 'hidden' class from the div with ui scanning makes the ui scanning show up again.

**Comment by hiukim** [comment]
fixed in v1.2.1
